portfolio/src/main/webapp/index.html

```diff
@@ -8,8 +8,22 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <div class="h1-container">
+        <h1>My Portfolio</h1>
+      </div>
       <p>This is my portfolio.</p>
+      <p>I love to learn whatever I can. From fitness, to economics, to software engineering, you have me on board</p>
+      <br/>
+      <h3>
+          Check out my portfolio below
+          <div class="upside-down">
```

Comment:
It's hard to tell without a demo to look at, but it seems like this image shouldn't be in the h3.  Is the title supposed to be on top of the image, or side-by-side?